server/core/services.py

```
@@ -143,6 +143,64 @@ def get_multiple_posts(request):
+            except Super.MultipleObjectsReturned:
+                return {"error": "Multiple miscs found with the given ID"}  # Handle duplicates properly
+            
+            
+
+            post = Post(
+                title = title,
+                user=user,
+                text=text,
+                image_url=image_url,
+                contentType=Post.PostType.TEXT if text else Post.PostType.IMAGE, 
+                project=None if not project else project,
```

Ruff (0.8.2)

Use `project if project else None` instead of `None if not project else project`

Replace with `project if project else None`

(SIM212)